Frends.Zip.CreateArchive/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.1.1] - 2024-12-13
+### Changed
+- Drop DotNetZip in favour of ProDotNetZip because of security reasons
```

Comment:
See the other changelog's codeRabbit comment and my addition to it - would bee nice to include this info here as well